app/views/brexit_checker/results.html.erb

```diff
@@ -18,7 +18,8 @@
   <link rel="canonical" href="<%= page_url %>">
 <% end %>
 
-<div class="govuk-width-container brexit-checker-results-page">
+<% content_for :breadcrumbs do %>
+<div class="govuk-width-container">
```

Comment:
Nitpick: we would normally indent the markup inside a `content_for` block.

Reply:
Thanks, have added that in the next commit, so that it's easier to see what has changed here